main.go

```diff
@@ -129,6 +129,7 @@ func router() *mux.Router {
 	router.Handle("/event/delete", alice.New(main.apiAuthMiddleware).ThenFunc(main.EventDeleteHandler))
 	router.Handle("/activist/list", alice.New(main.apiAuthMiddleware).ThenFunc(main.ActivistListHandler))
 	router.Handle("/activist/save", alice.New(main.apiAuthMiddleware).ThenFunc(main.ActivistSaveHandler))
+	router.Handle("/activist/suspend", alice.New(main.apiAuthMiddleware).ThenFunc(main.ActivistSuspendHandler))
```

Comment:
This is merely for clarification.. 

So the URI pattern of using something like `/activists/:activistId/suspend` isn't being used in this project?

Also in the past I've used a DELETE request method for these types of "soft deletes", and just toggled a field named "deleted" as a boolean or a date-time stamp.

Reply:
We're not using that pattern, but that may be a good idea to adopt later. I think it's built into the mux router that we're using.

Re: using DELETE, our API is an RPC style api, not REST (see https://apihandyman.io/do-you-really-know-why-you-prefer-rest-over-rpc/). There isn't a huge difference between them, I just prefer RPC style because it's more explicit what operations exist, and you don't need to worry about the semantics of different HTTP verbs.